packages/sdks/overrides/react-native/src/functions/transform-block.ts

```diff
@@ -0,0 +1,15 @@
+export function transformBlock(block: any) {
```

Comment:
Can you elaborate/add a comment about why this override is only for react-native and not for vue?

Reply:
it's this https://github.com/BuilderIO/builder/blob/native-pixel/packages/sdks/overrides/react-native/src/functions/transform-block.ts#L4:L4

I didn't put it at the top bc we may do other transforms, so that specific transform is specifically bc react native doesn't support DOM specific fields like `tagName` and `properties`, but any browser framework (e.g. Vue, etc) does

Reply:
I can elaborate on that comment if you have any suggestions on how it can be more clear

Reply:
Thanks, that context is useful!

> I didn't put it at the top bc we may do other transforms

Good call.

Just copy pasting your github comment above into the code would be great! I added a suggestion below.